mantle/platform/util.go

```diff
@@ -41,7 +41,7 @@ func Manhole(m Machine) error {
 	}
 
 	tstate, _ := terminal.MakeRaw(fd)
-	defer terminal.Restore(fd, tstate)
+	defer terminal.Restore(fd, tstate) //nolint
```

Comment:
The outer function does return an error, so the result of the deferred `Restore()` here can be bubbled up. This likely requires changing the outer function to return a named error value.

Reply:
Thank you for review and the great example. Updated the error.